src/metabase/public_settings.clj

```diff
@@ -32,12 +32,14 @@
   (when-let [varr (resolve 'metabase-enterprise.sso.integrations.sso-settings/other-sso-configured?)]
     (varr)))
 
-(defn- sso-configured?
-  "Any SSO provider is configured"
-  []
-  (or (google-auth-configured?)
-      (ldap-configured?)
-      (ee-sso-configured?)))
+(defsetting sso-configured?
```

Comment:
Put this into the setting, so FE could use this key to check if any SSO integration is configured. Previously I was using these 4 keys to determine if an SSO is configured.
- `google-auth-client-id`
- `ldap-enabled`
- `jwt-enabled`
- `saml-enabled`

They're used to show the active status in the following UI which is not quite obvious if they're fully configured or not, but I guess since FE have form validation built-in, so only the above keys (`xxx-enabled`) will only be true when they're fully configured.
![image](https://user-images.githubusercontent.com/1937582/176889123-94636434-054a-4e2a-a15c-cfb5d8e502c4.png)


But since BE already has this function and another similar setting named `other-sso-configured?`, I think I should just use this setting in the FE.